lib/awsutil/credentials.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/aws/aws-sdk-go/aws/arn"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/sts"
+	"os"
```

Comment:
Please preserve the separation between Go standard packages and other packages. I swept the whole codebase after feedback from developers at Uber. They use goimports to automatically fix this.

Reply:
ok I'll do that.